understanding/22/accessible-authentication-minimum.html

```diff
@@ -68,16 +68,17 @@ <h2>In brief</h2>
     <section id="intent">
         <h2>Intent of Accessible Authentication</h2>
 
-        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method to log in. Most Web sites rely on usernames and passwords for logging in. Memorizing or transcribing a username, password, or one-time verification code places a very high or impossible burden upon people with certain cognitive disabilities.</p>
+        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method for users to authenticate -- to prove they are authentic account holders. Most Web sites rely on usernames and passwords for logging in, the most common form of authentication. Yet memorizing a username and password places a very high or impossible burden upon people with certain cognitive disabilities, as do additional steps often added to authentication processes, such as the need to transcribe a one-time verification code or solve a puzzle.</p>
```

Comment:
wording suggestions.  take em or leave em.

```suggestion
        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method for users to authenticate -- to prove they are authentic account holders. As the most prevelant form of authentication, Web sites commonly rely on usernames and passwords to log in. However, memorizing a username and password places a very high or impossible burden upon people with certain cognitive disabilities, as do additional steps often added to authentication processes. For insance, the need to transcribe a one-time verification code or requiring a puzzle to be solved.</p>
```

Reply:
I stumbled over the "as do" and would suggest another wording:

> However, memorizing a username and password places a very high or impossible burden upon people with certain cognitive disabilities.  Additional steps also make authentication processes more difficult for these users, for instance, the requirement to transcribe a one-time verification code or to solve a puzzle.